applications/FluidDynamicsApplication/custom_utilities/fic_data.h

```diff
@@ -0,0 +1,109 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Jordi Cotela
+//
+
+
+#if !defined(KRATOS_SYMBOLIC_FIC_DATA_H)
+#define KRATOS_SYMBOLIC_FIC_DATA_H
```

Comment:
Minor comment, but I'd just use `KRATOS_FIC_DATA_H` (since this element is not symbolically-generated).